scripts/get_sys_observability_data.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
```

Comment:
we don't want the input to be provided in this form (i.e., do not use stdin to get input)...

Read the parameters from commandline...

`./get_sys_observability_data.sh --cluster default --container xyz --namespace abc --labels "xyz=abc, lmn=pqr"`